From f707511bfb2787bcc7cffef9965481a53b96b60b Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 3 Nov 2011 20:04:20 +0000
Subject: [PATCH 0089/1000] replacement for trace.dll, trchdl.dll and
 trcserv.exe: process writes traces to file, using same interface as trace.dll
 and trchdl.dll needs no trcserv.exe trace.dll trcHDL.dll traceNoConfig.dll
 trcHDLNoConfig.dll traces.lib tracehdls.lib traceCompletes.lib
 tracehdlCompletes.lib traceNoConfigs.lib tracehdlNoConfigs.lib
 traceNoConfigCompletes.lib tracehdlNoConfigCompletes.lib

git-svn-id: svn://localhost/SelfServiceCommon/trunk@206 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Trace/TestClients/makefile.mak            | 5 +++--
 .../Massai/cpp/Trace/src/TestTraceHdlLib.cpp             | 9 ++++++---
 2 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Trace/TestClients/makefile.mak b/SelfServiceCommon/Massai/cpp/Trace/TestClients/makefile.mak
index 9afd4d74..8f7ba716 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/TestClients/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/Trace/TestClients/makefile.mak
@@ -10,8 +10,8 @@
 
 MY_BIN1 =                $(_BIN)\TestTraceLib.exe
 
-#MY_BIN1_USER_LIBS =     $(LIB_TRACE) $(LIB_MSYSTOOLS) 
-MY_BIN1_USER_LIBS =      $(_LIB)\traceCompletes.lib $(LIB_MSYSTOOLS) 
+MY_BIN1_USER_LIBS =     $(LIB_TRACE) $(LIB_MSYSTOOLS) 
+#MY_BIN1_USER_LIBS =      $(_LIB)\traceCompletes.lib $(LIB_MSYSTOOLS) 
 
 MY_BIN1_OBJS =           $(_OBJ)\TestTraceLib.obj
 
@@ -20,6 +20,7 @@ MY_BIN1_OBJS =           $(_OBJ)\TestTraceLib.obj
 MY_BIN2 =                $(_BIN)\TestTraceHdlLib.exe
 
 MY_BIN2_USER_LIBS =      $(LIB_TRACEHDL) $(LIB_MSYSTOOLS) 
+#MY_BIN2_USER_LIBS =      $(_LIB)\trcHDLNoConfig.lib $(LIB_MSYSTOOLS) 
 
 MY_BIN2_OBJS =           $(_OBJ)\TestTraceHdlLib.obj
 
diff --git a/SelfServiceCommon/Massai/cpp/Trace/src/TestTraceHdlLib.cpp b/SelfServiceCommon/Massai/cpp/Trace/src/TestTraceHdlLib.cpp
index f2133059..4a0a7406 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/src/TestTraceHdlLib.cpp
+++ b/SelfServiceCommon/Massai/cpp/Trace/src/TestTraceHdlLib.cpp
@@ -32,10 +32,13 @@ int main(int argc, char** argv)
       buffer[i]= i;
     }
     MASSAI::localtrace::writeTraceText(MajorError_HERE, "dump");
-    for(int j= 0; j < 100000; j++)
+    for(int j= 0; j < 100; j++)
     {
-      writeTraceText(t1, MajorError_HERE, "");
-      writeTraceText(t2, MajorError_HERE, "");
+      //int t3= openTrace("T3");
+      writeTraceText(t1, MajorError_HERE, "T1");
+      writeTraceText(t2, MajorError_HERE, "T2");
+      // writeTraceText(t3, MajorError_HERE, "T3");
+      //closeTrace(t3);
     }
     MASSAI::localtrace::writeTraceText(MajorError_HERE, "close1");
     closeTrace(t1);
-- 
2.41.0.windows.1

